purchases/src/main/kotlin/com/revenuecat/purchases/common/OfferingParser.kt

```diff
@@ -70,7 +79,12 @@ internal abstract class OfferingParser {
             }
         }
 
-        return Offerings(offerings[currentOfferingID], offerings, placements)
+        return Offerings(
+            current = offerings[currentOfferingID]?.withPresentedContext(null, targeting),
```

Comment:
Is it possible to use both placements and targeting at the same time? Just making sure since this will override the placementId to null, if it has one.